### ISSUE TITLE
Notifications Not Updating On Top Or Bottom Bar

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#200]
The notifications are there on the notification page, but neither bar is updating and there is no popping noise to indicate a notification.
Cannot replicate issue, but was reported by Princess_Awinita & Shentino

**Comment by martindale** [comment]
When first viewing the notification, does it show as "read" already?  This might be due to notifications being automatically marked as read.

**Comment by Lady-Awinita** [comment]
> When first viewing the notification, does it show as "read" already? This might be due to notifications being automatically marked as read.

When viewing them no none are marked as read except for the last one read, the issue arises when the 0 next to the username does not update or rise in number

EDIT: This however works just fine on mobile devices {samsung tablet} but not on desktops or laptops

**Comment by martindale** [comment]
> > When first viewing the notification, does it show as "read" already? This might be due to notifications being automatically marked as read.
> 
> When viewing them no none are marked as read except for the last one read, the issue arises when the 0 next to the username does not update or rise in number
> 
> EDIT: This however works just fine on mobile devices {samsung tablet} but not on desktops or laptops

This indicates that notifications are being sent, but being marked as read by one of your other devices.  Can you confirm that all notifications do correctly show up in [your list of historical notifications](https://www.roleplaygateway.com/notifications)?  Are there any cases where a notification is not created?

**Comment by Lady-Awinita** [comment]
> > > When first viewing the notification, does it show as "read" already? This might be due to notifications being automatically marked as read.
> > 
> > 
> > When viewing them no none are marked as read except for the last one read, the issue arises when the 0 next to the username does not update or rise in number
> > EDIT: This however works just fine on mobile devices {samsung tablet} but not on desktops or laptops
> 
> This indicates that notifications are being sent, but being marked as read by one of your other devices. Can you confirm that all notifications do correctly show up in [your list of historical notifications](https://www.roleplaygateway.com/notifications)? Are there any cases where a notification is not created?

The notification is indeed being sent, I am not getting issues with notifications being sent out when I make posts, however. I have just checked both of my tablets, niether of them have RPG open in chrome so therefore none of my other devices are causing a already read issue.

However just now Nore has tagged one of my characters and the notice went through {I got the POP and the zero became a 1 until I accessed the notifications page, {where in the 1 became a 0 again as they had become read}}

**Comment by lostamongtrees** [comment]
Is this issue ongoing?

**Comment by Lady-Awinita** [comment]
> Is this issue ongoing?

Yes, I still am not receiving notifications on top or bottom bars, unless Rem tweaked something ?

**Comment by Lady-Awinita** [comment]
I have an update to this

Notifications ONLY show up for me when someone tags my characters via chat based posting using the explorer

**Comment by lostamongtrees** [comment]
Is this ongoing, or are you getting all of your notifications now @Lady-Awinita ?

**Comment by Lady-Awinita** [comment]
> Is this ongoing, or are you getting all of your notifications now @Lady-Awinita ?

I am getting notifications, but only when someone is using the excplorer, I am still testing this issue
